web/src/tippyjs.ts

```diff
@@ -426,7 +426,7 @@ export function initialize(): void {
     });
 
     delegate("body", {
-        target: "#email_invite_radio_container.disabled_setting_tooltip",
+        target: "[data-tab-key='invite-email-tab'].disabled-tab",
         content: $t({
             defaultMessage:
                 "You do not have permissions to send email invitations in this organization.",
```

Comment:
For consistency, let's make it "You do not have permissions to send invite emails in this organization."

Reply:
Done